src/style/search/search-bar.less

```diff
@@ -29,10 +29,11 @@
     background: no-repeat left @bg-panel;
     background-size: @text-xxlarge;
     background-position: 1.7rem 1.5rem;
-    top: 25%;
-    left: 50%;
+    top: round(25%);
```

Comment:
Agreed...this is a LESS function which doesn't seem to make it into the final CSS. It is translated to 25%...